user_docs/en/userGuide.t2t

```diff
@@ -2540,7 +2540,7 @@ Usually, this configuration should not be touched.
 To change NVDA's configuration during sign-in or on UAC screens, configure NVDA as you wish while signed into Windows, save the configuration, and then press the "use currently saved settings during sign-in and on secure screens" button in the General category of the [NVDA Settings #NVDASettings] dialog.
 
 + Add-ons and the Add-on Store +[AddonsManager]
-Add-ons are software packages, developed by the NVDA community and commercial vendors, which provide new or altered functionality for NVDA.
+Add-ons are software packages, developed by the NVDA community or by commercial vendors, which provide new or altered functionality for NVDA.

```

Comment:
Maybe this sentence should be split up as it's providing 2 different pieces of information.

```suggestion
Add-ons are software packages which provide new or altered functionality for NVDA.
They are developed by the NVDA community, and external organisations such as commercial vendors.
```